sstp_protocol/src/header.rs

```diff
@@ -0,0 +1,3 @@
+pub mod map;
```

Comment:
https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html#no-more-modrs
Isn't it old style to use `mod.rs`?
If it has changed since the 2018 style, can you give me a link for reference?

Reply:
Thank you!
I like the 2015 style too, but this repository will be in the 2018 style.